lunar/distribution.yaml

```diff
@@ -675,5 +675,21 @@ repositories:
       url: https://github.com/ros/std_msgs.git
       version: groovy-devel
     status: maintained
+  urdfdom_py:
+    doc:
+      type: git
+      url: https://github.com/ros/urdf_parser_py.git
+      version: 0.3.2
```

Comment:
No, bloom doesn't automatically update the doc entry when you do a release. Only the release entry. You could specify a branch instead if that contains the latest content.